### ISSUE TITLE
Coverage statistics

### ISSUE THREAD

**Issue by mjvankampen** [MicrosoftPremier/VstsExtensions#114]
Would it be possible to highlight coverage changes per file or something along those lines? I currently spend quite a bit of time hunting for the missing coverage. I currently just open the coverage pages of both builds and start scrolling to compare, quite tedious. Would be great to have this in BQC, similar to warnings statistics!

**Comment by mjvankampen** [comment]
Fair enough, I'll try tools like https://diff-cover.readthedocs.io/en/latest/README.html and cross my fingers for https://docs.microsoft.com/en-us/azure/devops/pipelines/test/codecoverage-for-pullrequests?view=azure-devops to implement cobertura

**Comment by ReneSchumacher** [comment]
It looks like you're using Cobertura for reporting coverage, right? This might be a good first start since the format of the Cobertura report is quite easy to understand, parse, and compare. I'm not sure how quickly we can add something but we'll make sure to start with Cobertura coverage ;-) Our own (i.e., Microsoft) coverage format is binary, thus, it'll be somewhat harder to work with that.

**Comment by stan-sz** [comment]
@mjvankampen CC for PRs now supports the binary format, if that helps: https://github.com/Microsoft/vstest/issues/981#issuecomment-973994625
